frontend/src/components/NavBar.js

```diff
@@ -69,21 +69,13 @@ class NavBar extends React.Component<{
     }
 
     fetchName() {
-        $.ajax({
-            url: '/rest/user',
-            dataType: 'json',
-            cache: false,
-            type: 'GET',
-            success: (data) => {
-                const userInfo = data.data.attributes;
-                userInfo.user_id = data.data.id;
-                this.fetchImg(userInfo.avatar_id);
-                this.setState({ userInfo: userInfo, newWorksheetName: `${userInfo.user_name}-` });
-            },
-            error: (xhr, status, err) => {
-                console.error(xhr.responseText);
-            },
-        });
+        const callback = (data) => {
+            const userInfo = data.data.attributes;
+            userInfo.user_id = data.data.id;
+            this.fetchImg(userInfo.avatar_id);
+            this.setState({ userInfo: userInfo, newWorksheetName: `${userInfo.user_name}-` });
```

Comment:
why was there a dash (`{userInfo.user_name}-`) in the first place?

Reply:
I was just refactoring the code, did not change anything functionality wise.

Reply:
I think this is the default name since by default we want the name of the new worksheet to be username-xxxx, so the user just need to enter what's after the dash.